mavenRepo/org/robobinding/aspectj-plugin/0.8.4/aspectj-plugin-0.8.4.jar.md5

```diff
@@ -1 +1 @@
-0d2d30bffed5714a3f623be3ac521e63
```

Comment:
@xudeheng , Is the new 0.8.5 ready? I think the contribution is very helpful for other people. Please let me know when you are ready.